src/seedu/addressbook/storage/jaxb/AdaptedPerson.java

```diff
@@ -52,15 +52,15 @@ public AdaptedPerson(ReadOnlyPerson source) {
 
         phone = new AdaptedContactDetail();
         phone.isPrivate = source.getPhone().isPrivate();
-        phone.value = source.getPhone().value;
+        phone.value = source.getPhone().toString();
 
         email = new AdaptedContactDetail();
         email.isPrivate = source.getEmail().isPrivate();
-        email.value = source.getEmail().value;
+        email.value = source.getEmail().toString();
 
         address = new AdaptedContactDetail();
         address.isPrivate = source.getAddress().isPrivate();
-        address.value = source.getAddress().value;
+        address.value = source.getAddress().toString();
```

Comment:
this commit doesn't seem to be related to this activity. try to keep your PR clean.